doc/howto/network_ovn_peers.md

```diff
@@ -5,34 +5,56 @@ discourse: 12165
 (network-ovn-peers)=
 # How to create peer routing relationships
 
-Network peers allow the creation of routing relationships between two OVN networks.
-This allows for traffic between those two networks to stay within the OVN subsystem rather than having to transit via the uplink network.
+By default, traffic between two OVN networks goes through the uplink network.
+This routing is inefficient, however, because it limits the network bandwidth to the uplink network's capabilities.
```

Comment:
> This path is inefficient, however, because it means that packets have to leave the OVN subsystem and transit via the host's networking stack (and potentially an external network) and back into the OVN subsystem of the target network. Depending on how the host's networking is configured this may also limit the bandwidth available if the OVN overlay network is on a higher bandwidth network that the host's external network.